dnsutil.go

```diff
@@ -313,12 +313,19 @@ func updateDomainWithCName(r *dns.Msg, fqdn string) string {
 }
 
 // recursiveNameservers are used to pre-check DNS propagation. It
-// prepends user-configured nameservers (custom) to the defaults
-// obtained from resolv.conf and defaultNameservers and ensures
-// that all server addresses have a port value.
+// picks user-configured nameservers (custom) OR the defaults
+// obtained from resolv.conf and defaultNameservers if none is
+// configured and ensures that all server addresses have a port value.
 func recursiveNameservers(custom []string) []string {
-	servers := append(custom, systemOrDefaultNameservers(defaultResolvConf, defaultNameservers)...)
+	var servers []string
+	if len(custom) == 0 {
+		servers = systemOrDefaultNameservers(defaultResolvConf, defaultNameservers)
+	} else {
+		servers = make([]string, len(custom))
+		copy(servers, custom)
+	}
 	populateNameserverPorts(servers)
+	fmt.Printf("populated servers, ignoring resolve.conf: %v", servers)
```

Comment:
Remove this; proper way to emit this in production would be a log, but we'd have to refactor for that. (not necessary for this PR IMO).

```suggestion
```

Reply:
fixed, sorry